.travis.yml

```diff
@@ -1,9 +1,16 @@
 language: node_js
+os:
+  - linux
+  - osx
 node_js:
-  - "0.10"
+  - "6"
+  - "5"
+  - "4"
   - "0.12"
-  - "iojs"
+  - "0.10"
 before_install:
   - npm update -g npm
 matrix:
   fast_finish: true
+  allow_failures:
```

Comment:
this is the only thing I have a problem with but the tests are passing so I'm going to merge anyway